debugger_protocol.rst

```diff
@@ -1100,6 +1102,11 @@ a *function* breakpoint.
 This command is used by the IDE to update one or more attributes of a
 breakpoint that was already set on the debugger engine via *breakpoint_set*.
 
+If a debugger engine supports the resolved state for breakpoints, the resolved
```

Comment:
The "0" is a left over mistake. I'll fix that to say "resolved" and "unresolved" here.

The rediscovery is something that the engine should do on its own. The breakpoint_get and breakpoint_list command should just return what the current state is - but there is of course no problem if a debugging engine elects to run this rediscovery at that stage too.

I'll see if I can improve the wording a little.